packages/iTwinUI-react/src/core/Wizard/Step.test.tsx

```diff
@@ -210,11 +212,57 @@ describe.each(['long', 'workflow'] as const)('Wizard step (%s)', (type) => {
         index={1}
         currentStepNumber={0}
         totalSteps={4}
-        type={type}
+        type={'long'}
       />,
     );
 
-    const step = container.querySelector('.iui-wizard-step') as HTMLElement;
+    const step = container.querySelector('.iui-stepper-step') as HTMLElement;
+    expect(step).toBeTruthy();
+    expect(step.style.width).toBeFalsy(); // not 25%
+  });
+});
+
+describe('Wizard step (workflow)', () => {
+  it('should render correctly', () => {
+    const step = (
+      <Step title='Second step' index={1} totalSteps={3} type={'workflow'} />
+    );
+
+    const { container } = render(step);
+
+    // Renders step
+    const stepContainer = container.querySelector(
+      '.iui-workflow-diagram-step',
+    ) as HTMLElement;
+    expect(stepContainer).toBeTruthy();
+
+    // No tabindex
+    expect(stepContainer).not.toHaveAttribute('tabindex');
+
+    // No title
+    expect(container.querySelector('.iui-stepper-step-name')).toBeFalsy();
+    // Content
+    const content = container.querySelector(
+      '.iui-workflow-diagram-content',
+    ) as HTMLElement;
+    expect(content).toBeTruthy();
+    expect(content.textContent).toBe('Second step');
+  });
+
+  it('should not set dynamic inline width', () => {
+    const { container } = render(
+      <Step
+        title='Mock step'
+        index={1}
+        currentStepNumber={0}
```

Comment:
Does `currentStepNumber` do anything on workflows?

Reply:
It does not. I have separated Step into StepperStep and WorkflowDiagramStep. WorkflowDiagramStep does not take `currentStepNumber` as a parameter anymore